src/MacSrc/OpenGL.cc

```diff
@@ -753,8 +757,12 @@ int opengl_bitmap(grs_bitmap *bm, int n, grs_vertex **vpl, grs_tmap_info *ti) {
         // recalculate it from the offset into the global lighting lookup
         // table.
         light = 1.0 - (ti->clut - grd_screen->ltab) / 4096.0f;
+
+        light *= light;
```

Comment:
Instead of squaring the light intensity here, it could be offloaded to the shader instead.